docs/usage.rst

```diff
@@ -1,4 +1,74 @@
 Usage
 =====
 
-TODO: Explain how to use your library.
+Prerequisites
+-------------
+
+To perform its function, django-critical-css requires:
+
+* `penthouse <https://www.npmjs.com/package/penthouse/>`_ service available.
```

Comment:
django-critical-css requires the service which we've build around the _penthouse_ package.

For that we need to do the following things:

1. Make https://github.com/moccu/penthouse-service repo public (+ update documentation?)
2. Publish that via NPM (+ update documentation)
3. Refer to docker repo: https://cloud.docker.com/u/moccu/repository/docker/moccu/penthouse-service